templates.rst

```diff
@@ -1553,23 +1553,18 @@ as currency:
     {# pass in the 3 optional arguments #}
     {{ product.price|price(2, ',', '.') }}
 
-Create a class that extends ``AbstractExtension`` and fill in the logic::
+.. _templates-twig-filter-attribute:
+
+Create a class and fill in the logic::
```

Comment:
```suggestion
Create a class with a method that contains the filter logic, then add
the `#[AsTwigFilter]` attribute to define the name and options of
the Twig filter::
```

Reply:
Added, but somehow Github doesn't want me to resolve the conversation :/